### PR TITLE
#9088 cleanup terminology

### PR DESCRIPTION
**Pull request by lars-erik**
Work to replace master with primary in the LB docs. Companion PR to https://github.com/umbraco/Umbraco-CMS/pull/9185.

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks for the PR @lars-erik !

I'll make sure this is reviewed as soon as possible.

File: Getting-Started/Setup/Server-Setup/Load-Balancing/index.md [hunk 1]
```diff
@@ -148,9 +148,13 @@ The reason you need a single server is because there is no way to guarantee tran
...
 
-Yes. There are no problems with having your master admin server also serve front-end request.
+Yes. There are no problems with having your primary admin server also serve front-end request.
 
 However, if you wish to have different security policies for your front-end servers and your back
 office servers, you may choose to not do this.
+
+__Question>__ _Whatever happened to Master?_
+
+Master, along with its less favorable sibling, is being removed from tech language. It's called Primary now. Move along.
```
Comment (marcemarc) [inline Getting-Started/Setup/Server-Setup/Load-Balancing/index.md:160]:
```suggestion
Master, along with its less favorable sibling, is being removed from tech language. The terms did not accurately describe the relationship between Umbraco instances in a Load Balancing scenario. We prefer to use Primary and Replica now. 
```

**Review by marcemarc — COMMENTED** [review]
Hi @lars-erik 

Thanks for taking a dive through the docs to make these updates, generally the flow seems to read fine, by just replacing the terms. I spotted a couple of typos (not created by you) that we should tweak at the same time, if that's ok? 

The only suggestion on content I've made is in answering the last question, I just thought it was good to make the point that the original terms, didn't really help in the understanding of the relationships of servers in an Umbraco load balancing setup! 

Thanks again for taking this on.

regards

Marc

**Comment by lars-erik** [comment]
Cheers, @marcemarc! 🍻

**Comment by sofietoft** [comment]
Hi @lars-erik !

It's been a while since this PR was opened, and as it doesn't look like the naming here is going to be changed in Umbraco 8 - one can hope for a change in v9 though 🤞

That's also why I'll close this PR for now.
We can always reopen it, and make the appropriate changes for the latest version if needed!

Thanks again for taking the time to update these articles!

**Pull request closed without merging**